src/actions/guides/deploying/heroku.cr

```diff
@@ -89,8 +96,7 @@ class Guides::Deploying::Heroku < GuideAction
     to our app:
 
     ```bash
-    $ heroku buildpacks:add
-    https://github.com/heroku/heroku-buildpack-nodejs
+    $ heroku buildpacks:add https://github.com/heroku/heroku-buildpack-nodejs
```

Comment:
Made this 1 line so it can be copied as is.

Reply:
Good idea. Also I don't think we need the `$`. I've always found that to be unhelpful since sometimes it makes it harder to copy. Up to you though. Not a huge deal either way :P

Reply:
I originally was going to remove it, but I saw we do it in so many places. I'll remove them all from this guide, and we can go through the other later to make it consistent.